### DISCUSSION TITLE
Question for monkeyman192

### DISCUSSION THREAD

**Discussion by philou-felin** [NMSCD/community#7] [Q&A] [answered]
@monkeyman192 ?

Do you remember how your "Working Portals" mods work? They are featured on https://nomansskyretro.com/mods.php and I plan to try them.

**Comment by monkeyman192** [comment, marked as answer]
This mod is deprecated because portals now work 😅
Maybe I should remove it to avoid confusion, but I basically added the interaction that was added at the time to make them do something.

**Reply by philou-felin** [reply]
I asked because I wanted to use it with those old versions. I found the answer on NexusMod. Thanks for the reply!
